frontend/components/welcomeHeader.jsx

```diff
@@ -17,27 +17,58 @@ const TitleContainer = styled.div`
 `;
 
 const TimerContainer = styled.div`
-  align-self: flex-end;
+`;
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TableContainer = styled.div`
+`;
+
+const SolvedContainer = styled.div`
 `;
 
 class WelcomeHeader extends React.Component {
-  render() {
-    const { gameState } = this.props;
+  constructor(props) {
+    super(props);
+    this.renderSolved = this.renderSolved.bind(this);
+  }
 
-    const { start_time } = gameState || {};
+  renderSolved(gameState) {
+    var solvedCounter = 0;
+    const gameStateValues = Object.values(gameState);
+    for (var entry in gameStateValues) {
+      if (gameStateValues[entry]['solved'] == true) {
```

Comment:
irrelevant with the above comment, but just to let you know, you should almost always use `===` instead of `==`. also since you're testing a boolean value, `=== true` isn't necessary